src/nixos-anywhere.sh

```diff
@@ -46,6 +46,8 @@ Options:
   URL of the source Nix store to copy the nixos and disko closure from
 * --build-on-remote
   build the closure on the remote machine instead of locally and copy-closuring it
+* --legacy-ssh
```

Comment:
Maybe:

```suggestion
* --builder-protocol
```

Than we can also support other protocols in future?